app/questionnaire/placeholder_renderer.py

```diff
@@ -112,3 +126,36 @@ def render(
             set_pointer(dict_to_render, pointer, rendered_text)
 
         return dict_to_render
+
+    def check_transform(self, transform: MutableMapping[str, Any]) -> bool:
+        if transform.get("transform") in self._placeholder_transforms:
+            if items := transform["arguments"].get("items"):
+                for item in items:
+                    if self.check_item(item):
+                        return True
+
+        return False
+
+    def check_item(self, item: MutableMapping[str, Any]) -> bool:
```

Comment:
Could we rename this specifically,  i.e
`is_item_on_routing_path()`

Also method could be private if we don't intend to use it outside of the placeholder renderer

Reply:
Updated and made private 👍